File: Lab3/Lab3_CPP/Lab3_CPP/Matrix.cpp
#include "Matrix.h"

int Matrix::GetElem(size_t i, size_t j)
{
	return this->elem[i][j];
}

void Matrix::SetMatrix(size_t row, rsize_t column)
{
	this->No_Rows = row;
	this->No_Columns = column;
}

void Matrix::SetMatrixItem(size_t row, size_t column, int item)
{
	this->elem[row][column] = item;
}

string Matrix::toString()
{
	string str = "";
	for (size_t i = 0; i < this->No_Rows; i++)
	{
		for (size_t j = 0; j < this->No_Columns; j++)
			str = str + " " + to_string(elem[i][j]);
		str = str + "\n";
	}
	return str;
}

File: Lab4/Lab4/Lab4/Model/Matrix.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab4.Model
{
    class Matrix
    {
        private int NoColumns;
        private int NoRows;
        private int[,] matrix;

        public Matrix(int NoRows, int NoColumns)
        {
            this.NoColumns = NoColumns;
            this.NoRows = NoRows;
            this.matrix = new int[this.NoRows, this.NoColumns];
        }

        public int GetNoColumns() { return this.NoColumns; }
        public int GetNoRows() { return this.NoRows; }
        public int GetMatrixItem(int row, int column) { return this.matrix[row, column]; }
        public void SetMatrixItem(int row, int column, int val) { this.matrix[row, column] = val; }
        public void SetNoColumns(int NoColumn) { this.NoColumns = NoColumn; }
        public void SetNoRows(int NoRow) { this.NoRows = NoRow; }

        public override string ToString()
        {
            string str = "";
            
            for (int i = 0; i < NoRows; i++)
            {
                for (int j = 0; j < NoColumns; j++)
                    str += matrix[i, j] + " ";
                str += "\n";
            }

            return str;
        }
    }
}
